### ISSUE TITLE
Chemistry in MathML needs to be identified in metadata

### ISSUE THREAD

**Issue by GeorgeKerscher** [w3c/a11y-discov-vocab#106]
On the User Experience Guide for Accessibility Metadata call, we discovered that in the EPUB Accessibility Metadata, we have no way to identify that the MathML markup is for Chemistry. ONIX has this ability.

To help, Neil writes:
For chemistry, there is a "chemical-element" property and there is a "chemical-formula" property. The latter might switch the speech for "=" from "equals" to "double bond". 

So, we think this functionality needs to be added. Once this is available, we can insert it to the UX Guide.

**Comment by mattgarrish** [comment]
What do you propose calling the property? ChemML and MathML are so close in naming that squashing them together doesn't yield any great names.

MathML4Chem, since it relies on MathML4? But maybe is a bit too punny.

**Comment by madeleinerothberg** [comment]
It sounds like this is not ChemML, which is a separate mark-up language and for which we already have a vocabulary entry. Instead, it is MathML being used for chemistry. How about "ChemistryInMathML"?

**Comment by clapierre** [comment]
I agree with Madeline here, although to be consistent with naming convention I think it should be "chemistryInMathML" with the initial c being lower case.

**Comment by mattgarrish** [comment]
It just sounds like we're starting to write descriptions instead of tokens. I don't know how we avoid abusing the camel-casing here, though.

**Comment by clapierre** [comment]
if we want to shorten this it could be "chemInMathML" I am fine with either.

**Comment by mattgarrish** [comment]
It's more the "in" descriptor in the middle. We're describing a flavour of something we already have properties to define two different cases for. If we were still using slashes, this would probably be MathML/chemistry.

It's not a major roadblock for me, but something about compressing a description doesn't quite click for me (btw, I don't really like my suggestion above, so not advocating for it.)

I sort of want something like chemMarkup and in the description we define that that means chem equations and formulas done in mathml rather than chemml, but even that's kind of confusing for the same reason we have two MLs in already.

Go with whatever you think is best. I've said my piece.
